packages/components/src/RadioGroup/RadioGroup.css

```diff
@@ -1,12 +1,12 @@
+.radioGroup {
+  display: inline-flex;
+  flex-direction: column;
+}
+
 .input {
   /* Hide checkbox on UI but not screen readers and still allow focus state */
   position: absolute;
-  width: 1px;
-  height: 1px;
-  overflow: hidden;
-  clip: rect(0 0 0 0);
-  clip-path: inset(100%);
-  white-space: nowrap;
+  left: -999em;
```

Comment:
Is there any chance this does any odd scrolling. Also, not problem with what you have here but I wonder if `vw` would be safer for those people with REALLY REALLY REALLY wide displays? :p

Reply:
Moving left off screen doesn't cause scroll. I'll make the `vw` change for those users with displays greater than 15984px lol.

Reply:
Not only do they need a window that wide but the checkbox would have to be _right_ on the right edge of the window. But, I think that we'll get a nice thank you letter from a wide screen user thanking you for this at some point. 😂